src/drive/web/modules/services/components/iconsContext.js

```diff
@@ -0,0 +1,9 @@
+const iconsContext = require.context(
```

Comment:
oups:
![Capture d’écran 2022-01-11 à 17 50 39](https://user-images.githubusercontent.com/8363334/148985964-1caf7ea3-50e0-4eaa-94d2-339251300097.png)

Reply:
I don't understand if I still need to read this PR or not with this screenshot?

Reply:
The PR was in Draft when you saw it. Now the error above is fixed in the PR and the PR is ready for review.